web/src/components/Accordion.tsx

```diff
@@ -46,12 +76,18 @@ export default function Accordion({
           {title}
         </h3>
         {badge}
-        <Icon
+        <AnimatedIcon
           className="text-black dark:text-white"
+          style={{ rotate: spring.rotate.to((r) => `${r}deg`) }}
           data-test-id={isCollapsed ? 'collapse-down' : 'collapse-up'}
         />
       </button>
-      {!isCollapsed && <div>{children}</div>}
+      <animated.div style={{ height: spring.height }}>
+        {/* The div below is used to measure the height of the children
+         * DO NOT REMOVE IT
+         */}
+        {renderChildren && <div ref={ref}>{children}</div>}
```

Comment:
An alternative to this would be to create a AccordionContent component that exposes a forward ref. That might be more future prof but will require us to change all the places where we use the accordion.